src/clarity-icons/clarity-icons.scss

```diff
@@ -95,11 +95,25 @@ clr-icon {
     .clr-i-badge {
         @include setIconColor($clr-icon-color-danger);
     }
-    &>svg {
+    & > * {
+        pointer-events: none;
+    }
+    & > svg {
         height: 100%;
         width: 100%;
         display: block;
         transition: inherit;
+        title {
+            display: none;
+        }
+    }
+    & > span[id^=_clr_icon_] {
+        position: absolute;
+        left: -10000px;
```

Comment:
This will cause scrollbars to appear if the display is changed from ltr to rtl. Could you reuse this class instead? https://github.com/vmware/clarity/blob/master/src/clarity-angular/popover/common/_popover.clarity.scss#L6